posthog/models/async_migration.py

```diff
@@ -49,6 +49,11 @@ class Meta:
 
     parameters: models.JSONField = models.JSONField(default=dict)
 
+    def __str__(self) -> str:
```

Comment:
maybe make it `get_version_requirements` or something like that, feels a bit odd for `__str__` to have this

Reply:
Fair point, updated.